controllers/feeds/getFeedChat.js

```diff
@@ -39,14 +39,22 @@ const mappingFeed = (req, feeds) => {
   return mapping;
 };
 
+const getMessage = (b) => {
+  const isImagesIncluded = b?.images_url?.length > 0 || b?.object?.images_url?.length > 0;
+  const isMessageEmpty = (b?.message || '')?.length === 0;
+  if (isImagesIncluded && isMessageEmpty) return 'Media 🏞️';
+  if (typeof b.object === 'object') return b?.object?.message;
+  return b?.message;
+};
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `getMessage` function seems to be handling the logic correctly. However, it's not clear what type of object `b` is expected to be. It would be beneficial to add some error handling or type checking to ensure that `b` is an object and has the expected properties before trying to access them.
```diff
+  if (!b || typeof b !== 'object') {
+    throw new Error('Invalid input for getMessage function');
+  }
   const isImagesIncluded = b?.images_url?.length > 0 || b?.object?.images_url?.length > 0;
   const isMessageEmpty = (b?.message || '')?.length === 0;
   if (isImagesIncluded && isMessageEmpty) return 'Media 🏞️';
   if (typeof b.object === 'object') return b?.object?.message;
   return b?.message;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->